core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMaterializedViewDefinition.java

```diff
@@ -163,10 +165,19 @@ public TypeId getType()
             return type;
         }
 
+        public Optional<String> getComment()
```

Comment:
Do we need to add `@JsonProperty` here? BTW the whole class is missing json annotations.